Formula/erlang@18.rb

```diff
@@ -0,0 +1,119 @@
+# Major releases of erlang should typically start out as separate formula in
+# Homebrew-versions, and only be merged to master when things like couchdb and
+# elixir are compatible.
+class ErlangAT18 < Formula
+  desc "Programming language for highly scalable real-time systems"
+  homepage "https://www.erlang.org/"
+  head "https://github.com/erlang/otp.git"
```

Comment:
Not sure we really need any head spec for erlang@18 but if we are going to have one it should be
```
  head "https://github.com/erlang/otp.git", :branch => "maint-18"
```

I suggest you just remove it, though.